docs/ui/auth_admin.rst

```diff
@@ -0,0 +1,16 @@
+.. _ref_ui_auth_admin:
+
+==========
+Auth Admin
```

Comment:
```suggestion
Auth admin
```

You can make this change globally, unless there's a case where a title is a proper name (e.g., `EdgeDB Cloud`). I don't think there are any cases of that here.

I think I've registered before that I don't love this convention we have of sentence case titles… but I love it slightly more for now than I would love a big PR to change them all to title case, so I guess that's where we are.